packages/data/src/subgraph.ts

```diff
@@ -1,6 +1,6 @@
 import { defaultNetwork, SupportedNetwork } from "@semaphore-protocol/utils/networks"
 import { AxiosRequestConfig } from "axios"
-import checkParameter from "./checkParameter"
+import { errorHandlers } from "@zk-kit/utils"
```

Comment:
```suggestion
import { requireString, requireObject, requireBoolean } from "@zk-kit/utils/error-handlers"
```

Similar to the `ethers.ts` file, you can use the `error-handlers` module directly.